remove_split_bitstreams.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+## This script removes files associated with a bitstream ID from the temporary web store.
+
+__author__ = 'daisie'
+
+import re
+import os
+import sys
+import subprocess
+
+WEB_ASSETS_BUCKET = "dryad-web-assets/temporary"
+
+def get_bitstream_id():
+    matched_bitstream_id = None
+    matchers = (
+       'http.*bitstream\/id\/(\d+)\/.*',
+       'http.*bitstreamID=(\d+).*',
+       '(\d+)',
+    )
+    while matched_bitstream_id is None:
+        raw = raw_input('Enter the bitstream ID or URL: ')
+        for matcher in matchers:
+            matched_bitstream_id = re.match(matcher, raw)
+            if matched_bitstream_id is not None:
+                break
+    return int(matched_bitstream_id.group(1))
+
+def main():
+    bitstream_id = None
+    if len(sys.argv) == 3:
```

Comment:
should this be "== 2" ? I don't see another argument. Please document the arguments in the comments at the beginning of the script.